src/h5web/providers/mock/utils.ts

```diff
@@ -76,6 +76,10 @@ export function makeStrAttr(name: string, value: string): HDF5Attribute {
   return makeAttr(name, { class: HDF5ShapeClass.Scalar }, stringType, value);
 }
 
+export function makeScalarIntAttr(name: string, value: number): HDF5Attribute {
```

Comment:
Would you mind replacing `Int` by `Integer` ? I had to read it twice the function name to understand it (and I got my coffee this time !)

Reply:
I was looking for consistency with `makeStrAttr`, which makes me realise that it should have been called `makeIntAttr`, but happy to change to `makeIntegerAttr`.